content/en/docs/project-user-guide/application-workloads/services.md

```diff
@@ -1,10 +1,167 @@
+
+## Service Type
+
+- **VirtualIP**：It is based on the unique IP generated by the cluster. The service inside the cluster can be accessed through this IP, and the service outside the cluster can be accessed through NodePort and LoadBalancer. This type is suitable for most services.
+- **Headless**：The cluster does not generate an IP for the service, and the service is directly accessed through the back-end Pod IP of the service within the cluster. This type is suitable for back-end heterogeneous services, such as services that need to distinguish between master and slave.
+
+{{< notice tip>}}
+
+In Kubesphere, the **Virtual IP** is a default type of a service that is created by **stateless** or **stateful**.<br />
+
+If you want to create a Headless service, you should use the mode of **YAML**.
+
+{{</ notice>}}
+
+## Prerequisites
+
+- You need to create a workspace, project and `project-regular` account. Please refer to the [Getting Started with Multi-tenant Management](../../../quick-start/create-workspace-and-project) if not yet.
+- You need to sign in with `project-admin` account and invite `project-regular` to enter the corresponding project if not yet. Please refer to [Invite Member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+## How to create a Service
+
+![create_service_type](/images/docs/project-user-guide/workloads/create_service_type.png)
+
+Create a service that has these methods in the Kubesphere: **Stateless Service**, **Stateful Service**, and **External Service**. what's more, you can custom creation through **Specify Workloads** and **Edit by YAML**.
+
+There are some ways to create a service
+
+- **Stateless Service**
+
+  Create a Stateless Service also create a **Deployment** workload.
+
+  For more information **Stateless**, please visit [Deployment](../../application-workloads/deployments)
+
+- **Stateful Service**
+
+  Create a Stateless Service also create a **Statefulset** workload.
+
+  For more information **Statefulset**, please visit [Statefulset](../../application-workloads/statefulsets)
+
+- **External Service**
+
+  The service created by External Service is a service internet access method. It is different from Stateless and stateful.
+
+- **Specify Workloads**
+
+  Create a service with existing Pods.
+
+- **Edit by YAML**
+
+  Create directly with Yaml, you can upload and download Yaml configuration files
+
+{{< notice tip>}}
+
+  The value of `annotations:kubesphere.io/serviceType` keywords is one of them **statelessservice** , **statefulservice** , **externalservice** and **None**.
+
+{{</ notice>}}
+
+## Create a **Stateless Service**
+
+Into **Application Workload**, click **Services**
+
+### Step 1: click the **Create**, choose **Stateless Service**
```

Comment:
### Step 1: click the **Create**, choose **Stateless Service**
->
### Step 1: click **Create**, and choose **Stateless Service**

Reply:
done